src/main/java/net/minecraftforge/event/entity/ProjectileImpactEvent.java

```diff
@@ -125,4 +126,23 @@ public FireworkRocketEntity getFireworkRocket()
             return fireworkRocket;
         }
     }
+    /**
```

Comment:
Please update these javadocs and remove the TODO. If the event is cancelled, then the raytrace hit for the fishing hook will not be processed and it will continue moving, similar to the other projectile impact events.

Reply:
Is it preferable to remove the doc line, as in most other ImpactEvents, or have it be the same as firework?

Reply:
Rewording the existing one for fireworks should be fine.